oauthproxy.go

```diff
@@ -743,11 +745,11 @@ func (p *OAuthProxy) UserInfo(rw http.ResponseWriter, req *http.Request) {
 
 // SignOut sends a response to clear the authentication cookie
 func (p *OAuthProxy) SignOut(rw http.ResponseWriter, req *http.Request) {
-	p.ClearSessionCookie(rw, req)
         //<!--COCO Begin-->
-        // COCO: calling CocostoreLogout to call coco logout API
+	p.ClearSessionCookie(rw, req)
+        // COCO: calling CocoLogout to call coco logout API
+        p.CocostoreLogout(rw, "logging out")
```

Comment:
- here the alignments are still not proper
- And, didnt i say the string as 'Signing out'? Why is this not per the comment?

Reply:
handled